w3-blob.md

```diff
@@ -523,106 +540,106 @@ Receipt MUST not have any effects.
 
 ## Location Commitment
 
-Location commitment represents commitment from the issuer to the audience that
-content matching the `content` [multihash] can be read via HTTP [range request]
+Location commitment represents a commitment from the issuer to the audience that content matching the `content` [multihash] can be read via a HTTP [range request].
 
-### Location Commitment Delegation Example
+### Location Commitment Capability
+
+Example:
 
-```js
+```jsonc
 {
-  "iss": "did:web:web3.storage",
+  "iss": "did:key:zStorageProvider",
   "aud": "did:key:zAlice",
-
-  "cmd": "/assert/location",
-  "sub": "did:web:web3.storage",
-  "pol": [
-    // multihash must match be for the blob uploaded
-    ["==", ".content", { "/": { "bytes": "mEi...sfKg" } }],
-    // must be available from this url
-    ["==", ".url", "https://w3s.link/ipfs/bafk...7fi"],
-    // from this range
-    ["==", ".range[0]", 0],
-    ["==", ".range[1]", 2_097_152],
-  ],
+  "att": [{
+    "can": "assert/location",
+    "with": "did:key:zStorageProvider",
+    "nb": {
+      // space the content was added to
+      "space": { "/": { "bytes": "did:key:zAlice" } },
```

Comment:
q: why do we need to use `Bytes<SpaceDID>` here instead of just `SpaceDID` as usual?

Reply:
In UCAN 0.9, `with` is a URI, so the DID we use here MUST be a string.

UCAN actually encodes `iss` and `aud` DIDs as bytes, so this is consistent with other DIDs encoded in the UCAN. When we implement UCAN 1.0 it'll be consistent with subject `sub` (what `with` becomes).